contrib/junit.tpl

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+<testsuites>
+{{ range . }}
+	{{- $failures := 0 }}
+    {{- range .Vulnerabilities -}}
+		{{- $failures = sum $failures 2 -}}
+	{{- end -}}
+	<testsuite tests="1" failures="{{ $failures }}" time="" name="{{  .Target }}">
```

Comment:
I'm not so familiar with JUnit, so I just want to ask you. Isn't it weird that tests=1 and failures=10? I'm guessing "tests" is always greater than "failures".

Reply:
Here in the template, I assumed that 1 test per image and each failure are reported against the one test ran on image

Reply:
@knqyf263 Updated the Readme and indentation, let me know if you want to update the tests format

Reply:
It sounds fine. Let's keep it as is.